### ISSUE TITLE
[Bug] Download button still shows when hide download button setting enabled

### ISSUE THREAD

**Issue by ego-lay-atman-bay** [arichorn/uYouPlusExtra#158]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Have you read the FAQ?

- [X] Yes, I did read the FAQ

### Device info

- iOS/iPadOS version: 15.3.1
- Device model: ipad pro (10.5 inch)
- Sideload tool (AltStore, Sideloadly, TrollStore,...): trollstore
- The specific version of uYouPlus (**latest or newest is NOT a version number!**): 18.40.1-3.0.1

### Describe the bug

The download button under the player still shows when the setting to remove it is turned on.

### Steps to reproduce the issue

1. Enable uYouPlus > Miscellaneous > Hide the Download Button under player
2. Go to a video
3. See download button
![4B37EA6E-5A17-4270-B2EE-E57A37B43B30](https://github.com/arichorn/uYouPlusExtra/assets/70973578/add7d7ed-5bea-4e47-950e-1d4015f01eec)
![E79AE820-80BD-4B6B-8FB5-57C145889061](https://github.com/arichorn/uYouPlusExtra/assets/70973578/97049ac8-8c8a-4681-ae1f-d9d32a7a19a3)

### Are you using the newest version of uYouPlus? If not, why?

✅ Yes, I'm using the latest version of uYouPlus right now

### Does the issue happen with the official YouTube from AppStore?

❌ No, I can't reproduce this issue in the official YouTube from AppStore

### Additional context

I also have noMosernUi turned off, which I would turn on if it supported the zoom feature.

**Comment by xhfpsxm** [comment]
> Perfect timing, I’m already working on a updated version of it.

Could you add other options (remix, thanks and clip) as well?

**Comment by Hex34568** [comment]
> ### Is there an existing issue for this?
> * [x]  I have searched the existing issues
> 
> ### Have you read the FAQ?
> * [x]  Yes, I did read the FAQ
> 
> ### Device info
> * iOS/iPadOS version: 15.3.1
> * Device model: ipad pro (10.5 inch)
> * Sideload tool (AltStore, Sideloadly, TrollStore,...): trollstore
> * The specific version of uYouPlus (**latest or newest is NOT a version number!**): 18.40.1-3.0.1
> 
> ### Describe the bug
> The download button under the player still shows when the setting to remove it is turned on.
> 
> ### Steps to reproduce the issue
> 1. Enable uYouPlus > Miscellaneous > Hide the Download Button under player
> 2. Go to a video
> 3. See download button
>    ![4B37EA6E-5A17-4270-B2EE-E57A37B43B30](https://user-images.githubusercontent.com/70973578/276518038-add7d7ed-5bea-4e47-950e-1d4015f01eec.jpeg)
>    ![E79AE820-80BD-4B6B-8FB5-57C145889061](https://user-images.githubusercontent.com/70973578/276518054-97049ac8-8c8a-4681-ae1f-d9d32a7a19a3.jpeg)
> 
> ### Crash log (if the app crashes)
> _No response_
> 
> ### Are you using the newest version of uYouPlus? If not, why?
> ✅ Yes, I'm using the latest version of uYouPlus right now
> 
> ### Does the issue happen with the official YouTube from AppStore?
> ❌ No, I can't reproduce this issue in the official YouTube from AppStore
> 
> ### Additional context
> I also have noMosernUi turned off, which I would turn on if it supported the zoom feature.

DUDE, this bug OMG has been around aince qnblack added the feature and still not fixed to this day jeez, atleast arichorn is working on it now

**Comment by ego-lay-atman-bay** [comment]
Yeah, I just never noticed it before (I honestly don't pay attention to the buttons under the player), or maybe it worked with the noModernUi setting.

**Comment by KieranBLK** [comment]
This option works as expected for me on the latest build. Just needs support for the rest of the buttons now. 👌

![image](https://github.com/arichorn/uYouPlusExtra/assets/120525685/b676ce3b-04c1-4c89-ae71-7a22010a4aab)

**Comment by ego-lay-atman-bay** [comment]
It doesn't work for me

**Comment by KieranBLK** [comment]
> Wait it’s working right now? I’m currently trying to fix it?

Doesn't seem to be consistent. It works for some videos but not most. Sorry for the confusion, it seemed like it was working perfectly.
